Cargo.toml

```diff
@@ -67,7 +67,7 @@ pest_derive = "2.6.0"
 quick-xml = "0.28.2"
 rand = "0.8.5"
 rayon = "1.7.0"
-regex = "1.8.1"
+regex = { version = "1.8.1", default-features = false, features = ["perf", "std", "unicode-perl"] }
```

Comment:
Is there a reason you have kept `unicode-perl` enabled, unlike the other unicode features?

Reply:
Unfortunately, this is needed even for `\d`, `\w` etc. that are used in starship code.

Reply:
The docs hint at that, but is that really true? The tests run fine without `unicode-perl`.